tests/ExamplesTest.php

```diff
@@ -360,6 +360,10 @@ public function testQueryableEncryptionExamples(string $file, string $expectedOu
 
         $this->skipIfServerVersion('<', '7.0.0', 'Queryable encryption tests require MongoDB 7.0 or later');
 
+        if ($this->isStandalone()) {
+            $this->markTestSkipped('Encrypted collections are not supported on standalone');
```

Comment:
```suggestion
            $this->markTestSkipped('Queryable encryption requires replica sets');
```

Optional suggestion if you want to use the same error message I did in https://github.com/mongodb/mongo-php-library/pull/1133 (in more places).